apps/server/src/lib/driver/google.ts

```
@@ -1193,4 +1231,36 @@ export class GoogleMailManager implements MailManager {
+      return labelName;
+    }
+
+    if (this.labelIdCache[labelName]) {
+      return this.labelIdCache[labelName];
+    }
+
+    const userLabels = await this.getUserLabels();
+    const existing = userLabels.find(
+      (l) => l.name?.toLowerCase() === labelName.toLowerCase(),
+    );
+    if (existing && existing.id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)